pkg/config/node.go

```diff
@@ -222,6 +303,16 @@ func GetConfig(kubeconfigPath, clusterConfigPath, resolvConfPath string, apiVip
 		}
 	}
 
+	node.BootstrapIP, err = getBootstrapIP(apiVip.String())
+	if err != nil {
+		log.Warn(err)
+	}
+
+	node.IngressConfig, err = GetIngressConfig(kubeconfigPath)
+	if err != nil {
+		log.Warn(err)
+	}
+
```

Comment:
1. getBootstrapIP & GetIngressConfig functions required only for keepalived-monitor, maybe we should consider split them into separate function or add a boolean flag to Getconfig function.
2. Another point, there's no meaning calling to 'getBootstrapIP' every poll interval forever after bootstrap was deleted. 
One side effect of that is keepalived-moniotor log is full with:
"time="2020-03-03T11:11:56Z" level=info msg="An error occurred on dial: dial tcp 192.168.111.5:64444: connect: connection refused"
time="2020-03-03T11:11:56Z" level=warning msg="dial tcp 192.168.111.5:64444: connect: connection refused" "

Reply:
1. Fixed.
2. I'll try to figure out a way to reliably detect that the bootstrap is no more. For now, I'd like to get this landed.